lib/masamune/transform/postgres/snapshot_dimension.psql.erb

```diff
@@ -28,11 +28,11 @@ http://www.postgresql.org/message-id/CAGnEbohhKmW55oB0FpQd3naXBkwi74E=8DRZBGFjS-
 WITH ranges AS (
   SELECT *,
   CASE WHEN delta = 0
-  THEN 1 ELSE NULL END r
+  THEN 1 ELSE 0 END r
```

Comment:
The CTE `ranges` is used to collect delta records together with their associated complete record denoted by `window_id`
This code assumes that a complete record will always be available before a delta record.  If this assumption is violated, this results in a `NULL` `window_id` for delta records that happen before complete records